### DISCUSSION TITLE
OpenRport Changelog

### DISCUSSION THREAD

**Discussion by alhdo** [openrport/openrport#3] [Announcements]
📝 Version 0.9.12 (2023-10-01)

🚀 We've got some  changes for you in this update! Check out what's new in OpenRPort:

Features
🧩 Built the Last Known Public Version: We've assembled the latest public version of RPort, ensuring a rock-solid foundation for further developments.

📦 Package Name Update: To reflect our evolution, we've renamed the package to "openrport." This paves the way for a fresh start and a clear identity.

🤝 Pairing Service URL: We're thrilled to introduce the all-new Pairing Service, now available at [https://pairing.openrport.io](https://pairing.openrport.io). Seamlessly connect to your RPort client!

🛠️ Installation Script Enhancement: Our installation script is now enhanced. It points directly to the latest binary built on GitHub, making the setup process a breeze.

📂 Installation Script URL: Get your hands on the installation script at [https://get.openrport.io](https://get.openrport.io). It's easier than ever to get started with RPort.

📚 Readme Update: We've updated a section of the readme to reference OpenRPort instead of RPort, making it crystal clear where to find all the latest and greatest information.

🎉 Thank you for being part of the RPort community, and we can't wait to see what's in store for OpenRPort!
Got questions or feedback? Feel free to reach out and share your thoughts. Happy coding! 🚀🌟

**Comment by darkleono** [comment]
Thanks a lot for all your effort. We love you guys!!!
